### ISSUE TITLE
[jibGradle] fatal error when ctrl-c-ing `skaffold-dev`

### ISSUE THREAD

**Issue by dgageot** [GoogleContainerTools/skaffold#1198]
When I interrupt `skaffold dev` by pressing ctrl-c, deployments are deleted as expected but the skaffold process ends with a fatal error related to jib:

```
^CCleaning up...
service "db" deleted
deployment.apps "db" deleted
service "game" deleted
deployment.apps "game" deleted
service "ref" deleted
deployment.apps "ref" deleted
service "ui" deleted
deployment.apps "ui" deleted
Cleanup complete in 5.390827535s
FATA[0057] listing files: listing files: getting jibGradle dependencies: Running [/Users/dgageot/src/tools/hexagons/game/gradlew _jibSkaffoldFiles -q]: stdout , stderr: , err: signal: killed: signal: killed
```

**Comment by dgageot** [comment]
ping @GoogleContainerTools/java-tools

**Comment by briandealwis** [comment]
I guess there was a `_jibSkaffoldFiles` running, and so it was killed off.  This is likely "fixed" by #1197 / 
https://github.com/GoogleContainerTools/jib/pull/1189 in that the `_jibSkaffoldFiles` will complete more quickly.

**Comment by briandealwis** [comment]
Would it make sense to use something like the [`kubernetes.IsTerminatedError()`](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/kubernetes/port_forward.go#L211) to return `nil`?
